### DISCUSSION TITLE
How to prevent links being underlined

### DISCUSSION THREAD

**Discussion by amdramsteve** [templaza/astroid-framework#384] [Q&A]
Hi
I've just started using Astroid as a replacement for Helix Ultimate on my websites. I'm finding it much easier to use and with several very useful features. Helix is a good framework but I found it irritating sometimes when small CSS changes wouldn't work. And my websites seem to load more quickly with Astroid. 
One minor thing which I can't resolve - how can I stop Astroid from underlining all my links? I prefer not to see underlined links but I can't find any settings in the framework dashboard and I've tried adding some simple custom CSS such as a {text-decoration: none;} but it doesn't work.
Any suggestions?

**Comment by amdramsteve** [comment]
Forget the above... I've just discovered that it was my page builder (Page Builder CK) causing the underlining of links. I can switch it off for each element but the default is ON. The underlining wasn't there using the Helix template, so I must assume Helix overrides it somewhere. 
Perhaps this could be a minor enhancement to Astroid? Ability to globally switch on or off underlining of links??

**Reply by sonvnn** [reply]
You can use custom css to override it. 
```
a {
  text-decoration: none;
}
```

To learn more about Custom css please read https://docs.astroidframe.work/custom-code/custom-css

Thanks & Best Regards,
Sonny
